app/controllers/games.rb

```diff
@@ -16,6 +17,7 @@
 
   game = Game.find_by(id: params[:id])
 
+  #ZM: SEE PR
```

Comment:
In the Post method down on guesses... Can you think of a better way rather then assigning each local variable directly?